packages/api-db-mongodb/src/db/schema.ts

```diff
@@ -108,9 +108,9 @@ export interface DBUser {
   email: string
   emailVerifiedAt: Date | null
   name: string
-  firstName?: string
-  preferredName?: string
-  address?: UserAddress
+  firstName?: string | null
```

Comment:
This is because Prisma will always return a property as null when it's not set in the database, while MongoDB will not include it (as it doesn't know about it). Everything was typed to also be nullable so that the typings are compatible until they are removed.

Reply:
For 

> How will we provide compatibility within the peering network when nodes are using different We.Publish versions? 

I currently still send the old args to the peerArticles alongside the new ones. I could successfully get Articles from demo, but I haven't done much more testing than that yet.